### ISSUE TITLE
ImportError: DLL load failed: The data present in the reparse point buffer is invalid.

### ISSUE THREAD

**Issue by MikhailMashukov** [scipy/scipy#13542]
I installed Python in Windows 7 virtual machine to a folder on a host machine. So my H drive on virtual machine is mapped to \\vmware-host and Python installed to H:\E\Programs\-Win7VM\Python35 (github hided \ between Programs and -Win7VM). I suspect this is the reason. But before scipy Python worked ok (only a bit slow and with requests about running from network drive), you can see some other installed libraries below.

Also copied it to C:\Program Files\Python35 and changed path in PATH, this didn't help. With administrator priviledges/without - not important. Didn't check on newer versions.

#### Reproducing code example:
pip.exe install scipy==1.3.2

example.py:
import scipy 

#### Error message:
C:\WorkDir>H:\E\Programs\-Win7VM\Python35\python.exe example.py
Traceback (most recent call last):
  File "example.py", line 5, in <module>
    import scipy.stats
  File "H:\E\Programs\-Win7VM\Python35\lib\site-packages\scipy\stats\__init__.py", line 379, in <module>
    from .stats import *
  File "H:\E\Programs\-Win7VM\Python35\lib\site-packages\scipy\stats\stats.py", line 180, in <module>
    import scipy.special as special
  File "H:\E\Programs\-Win7VM\Python35\lib\site-packages\scipy\special\__init__.py", line 641, in <module>
    from ._ufuncs import *
ImportError: DLL load failed: The data present in the reparse point buffer is invalid.

```

#### Scipy/Numpy/Python version information:
Windows 7 x64 in VMWare Player. Python 3.5.2.

llvmlite==0.31.0
matplotlib==2.0.0
numba==0.47.0
numpy==1.18.0
pyparsing==2.4.7
python-dateutil==2.8.1
pytz==2020.1
pywin32==300
scipy==1.3.2
six==1.15.0

**Comment by MikhailMashukov** [comment]
Reinstalling to C:\Python35 didn't help!

**Comment by andyfaff** [comment]
Sorry, it's a little hard to follow your issue. The first thing I would point out is that scipy and numpy are no longer made for python 3.5 (or 3.6). once you've installed a more modern I suggest making and activating a venv for installing packages into.

**Comment by rgommers** [comment]
This is very old (about scipy 1.4.x) and without enough info to be actionable; does not look like a bug in scipy either way. So I'll close this issue. Thanks everyone
